src/test/operators/print_test.cpp

```diff
@@ -123,4 +124,19 @@ TEST_F(OperatorsPrintTest, OperatorName) {
   EXPECT_EQ(pr->name(), "Print");
 }
 
+TEST_F(OperatorsPrintTest, TruncateLongValue) {
+  auto tab = StorageManager::get().get_table(table_name);
+
+  tab->append({0, "abcdefghijklmnopqrstuvwxyz"});
+
+  auto wrap = std::make_shared<TableWrapper>(tab);
+  wrap->execute();
+
+  auto printer = std::make_shared<Print>(wrap, output);
+  printer->execute();
+
+  auto output_str = output.str();
+  EXPECT_TRUE(output_str.find("abcdefghijklmnopq...") != std::string::npos);
```

Comment:
Could you explicitly test the new truncate method and use _col_width instead of hard-coding it?